tests/inc/test-class-addon-manager.php

```diff
@@ -344,6 +349,72 @@ public function test_get_plugin_information( $action, $args, $expected, $message
 		);
 	}
 
+	/**
+	 * Tests the validation of a valid subscription.
+	 *
+	 * @covers WPSEO_Addon_Manager::has_valid_subscription
+	 */
+	public function test_has_valid_subscription() {
+		$instance = $this
+			->getMockBuilder( 'WPSEO_Addon_Manager' )
+			->setMethods( array( 'get_subscriptions' ) )
+			->getMock();
+
+		$instance
+			->expects( $this->any() )
```

Comment:
Why any()?

Reply:
I think you are right in that it could be `once` in this case. Not sure why I did this anymore, these tests got altered a few times.
Same for the others.